Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -145,33 +145,34 @@ class DataDecoderTask
   }
 
  private:
-  std::string readFileContent(std::string_view filename)
+  std::string readFileContent(std::string& filename)
```

Comment:
why the change from string_view to (non-const) string reference (which seems wrong) ?

Reply:
The code was not compiling when I tested it on our reference FLP. I will check again after a full O2Suite update...

Reply:
no, no need, string_view is indeed wrong in that context. Noticed it in #3461 also and then I went to const std::string& (and forced-push)